crates/uv-python/src/installation.rs

```diff
@@ -390,10 +390,20 @@ impl PythonInstallationKey {
         &self.variant
     }
 
+    fn executable_base_name(&self) -> &'static str {
+        match self.implementation() {
+            LenientImplementationName::Known(ImplementationName::PyPy) => "pypy",
+            LenientImplementationName::Known(ImplementationName::GraalPy) => "graalpy",
+            LenientImplementationName::Known(ImplementationName::CPython) => "python",
+            LenientImplementationName::Unknown(_) => "python",
+        }
```

Comment:
FYI this duplicates some code found in `ManagedPythonInstallation::executable`: https://github.com/astral-sh/uv/blob/a82c210cabde1bb4422639d93ae2791a827c0bc9/crates/uv-python/src/managed.rs#L365-L369

Reply:
That other code seems wrong, I think we should just do https://github.com/astral-sh/uv/pull/14337

Here, I think we should probably do the same thing and make use of https://github.com/astral-sh/uv/blob/2966471db2dfafa71230cdedfa7f0cd124de02b7/crates/uv-python/src/implementation.rs#L74-L81

Reply:
Oh funny, that doesn't work because of `cpython`, I guess we need a special case for that 😭